packages/israel/israel.svg

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created for MapSVG plugin: http://mapsvg.com -->
+<svg
+   xmlns:mapsvg="http://mapsvg.com"
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   mapsvg:geoViewBox="34.228663 33.434207 35.935383 29.496766"
+   width="294.62534"
+   height="792.60406">
+ 
+  <path
+     d="m 290.6293,111.87855 -3.12,5.45 -2.53,5.56 -2.8,3 -4.23,7.59 -5.38,6.25 -0.16,-0.68 -1.09,-0.62 -3.76,1.55 -1.79,0.22 -0.86,0.73 -0.67,1.52 -0.7,2.77 -3.93,0.54 -2.48,2.51 -2.74,1.94 -1.71,0.52 -1.23,0.8 -4.27,0.3 -2.06,1.33 0,0 -0.09,-1.28 -0.49,-0.54 -3.3,-1.49 -1.25,-0.89 0.05,-0.4 3.27,-3.32 1.93,-4.04 0.53,-3.58 0.31,-7.42 1.21,-4.61 0.33,-3.08 -0.47,-3.25 -0.75,-1.94 -2.24,-4.39 -1.52,-1.63 -0.72,-1.24 -1.29,0.56 -0.79,-0.44 -0.36,-0.87 0.05,-1.06 1,-2.21 0.35,-1.75 0.76,-10.480002 -0.29,-4.73 -1.99,-3.74 -0.06,-0.96 2.83,-2.83 1.78,-3.4 0.81,-0.94 1.85,-0.56 0.5,0.58 0.06,-5.32 0.47,-4.48 2.57,-6.86 0.64,-2.49 0.01,-1.38 -0.83,-3.57 -0.31,-2.73 0.12,-4.47 0.84,-3.21 1.48,-3.42 -0.16,-0.53 -0.99,-0.58 -2.17,-0.29 -1.03,0.13 -2.26,1.21 -1.29,0.33 -1.28,-0.31 -1.44,-1.35 0,0 0.65,-1.63 0.81,-1.06 5,-2.35 5.27,-3.16 1.78,-1.71 2.56,-4.09 1.43,-1.3 1.07,-0.42 4.33,0.37 2.62,-0.45 3.46,-1.44 3.2,-2.61 1.01,-1.38 0.54,-1.62 0.8,-5.2299999 1.48,-2.36 2.62,-1.91 9.49,-5.26 0.89,0.69 0.24,1.47 -0.62,2.62 -1.8,2.98 -3.43,3.9099999 -1.47,3.82 -0.53,3.71 0.83,8.33 -0.26,1.4 -1.98,3.38 1.28,1.54 2.54,2.17 1.12,1.56 0.57,0.99 1.16,4.17 0.22,2.41 -0.34,1.63 -0.65,1.67 -1.54,1.68 -0.55,1.1 0.33,2.22 1.43,0.51 2.36,0.25 1.31,0.61 3.64,5.26 -0.41,5.37 -2.76,2.44 -0.32,0.85 0.39,1.12 1.95,2.21 0.2,0.88 -0.25,0.5 -0.92,0.55 -1.94,0.23 -3.43,-0.91 -0.73,0.59 -0.41,1.07 0.87,3.58 -0.51,6.44 0.9,2.93 1.48,13.610002 3,4.16 0.59,1.43 0.01,1.13 -0.79,2.18 z"
+     title="Golan Heights"
+     id="IL-GH" />
```

Comment:
Could you use more explicit `id`s  please? For example: `golan-heights`, `hazafon`...